wiki/en/misc/general.yml

```diff
@@ -22,3 +22,4 @@ kb:
   discuss:
     title:           "Comments"
     linktitle:       "Discuss this content"
+infobox_each_os:     'Jamulus 3.9.1 addresses a MIDI controller-related bug that could result in the loss of old fader levels: If you used `--ctrlmidich`, you can start the app once with `--cleanuplegacyfadersettings` to try to convert the old format to the new one. This option will be removed in a future release. Read the <a href="https://github.com/jamulussoftware/jamulus/blob/master/ChangeLog">Change Log</a> for more information.'
```

Comment:
Will this get picked up for translation?  Isn't it simpler just to put the content into `_includes/infobox_each_os.html`?

Reply:
It will. And it's much easier than writing .po(t) files for _includes, I think.

Reply:
Maybe have it as a yaml multiline string, then.  Like this looks quite awkward.

Reply:
As almost nobody looks at it here anyway, I think it's easier to have a long string in here.